test/components/ChangeNameMutation.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { graphql, commitMutation } from 'react-relay';
+import { Environment } from 'relay-runtime';
+
+const mutation = graphql`
+mutation ChangeNameMutation($input: ChangeNameInput!) {
+  ChangeName(input: $input) {
+    me {
+      id
+      name
+    }
+  }
+}
+`;
+
+let tempID = 0;
+
+const commit = (environment: Environment, name: string) => {
```

Comment:
Pass environment directly?

Reply:
We don't have a way to import directly the previous created environment(with all our config) yet :(